go/integTest/cluster_commands_test.go

```diff
@@ -1780,3 +1781,245 @@ func (suite *GlideTestSuite) TestFunctionStatsWithRoute() {
 		assert.Equal(t, int64(0), nodeStats.Engines["LUA"].LibraryCount)
 	}
 }
+
+func (suite *GlideTestSuite) TestFunctionKillNoWriteWithoutRoute() {
+	suite.skipOnVersionLowerThan("7.0.0", "This feature added in version 7")
+
+	client := suite.defaultClusterClient()
+	libName := "functionKill_no_write_without_route"
+	funcName := "deadlock_without_route"
+	code := createLuaLibWithLongRunningFunction(libName, funcName, 6, true)
+
+	// Flush before setup
+	result, err := client.FunctionFlushSync()
+	assert.NoError(suite.T(), err)
+	assert.Equal(suite.T(), "OK", result)
+
+	// Nothing loaded, nothing to kill
+	_, err = client.FunctionKill()
+	assert.Error(suite.T(), err)
+	assert.True(suite.T(), strings.Contains(strings.ToLower(err.Error()), "notbusy"))
+
+	// Load the lib
```

Comment:
As we discussed, we can complete the test at that place. All the rest could be moved to IT++ suite.
It is a subject of another PR to create that suite and alter CI and makefile.
After that we can add those tricky tests there.

Reply:
Applicable to all your test